app/src/main/java/com/example/houzboat/MapsActivity.kt

```diff
@@ -36,8 +36,8 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
         mMap = googleMap
 
         // Add a marker in Sydney and move the camera
-        val sydney = LatLng(-34.0, 151.0)
-        mMap.addMarker(MarkerOptions().position(sydney).title("Marker in Sydney"))
+        val sydney = LatLng(-35.5, 151.0)
+        mMap.addMarker(MarkerOptions().position(sydney).title("Marker is Sidney"))
```

Comment:
I'll correct the name Sydney
```suggestion
        mMap.addMarker(MarkerOptions().position(sydney).title("Marker is Sydney"))
```